app/serializers/avatar_obj_serializer.rb

```diff
@@ -1,5 +1,7 @@
 class AvatarObjSerializer < BaseSerializer
   attribute :avatar do
-    avatar_href_link(object)
+    uri = nil
+    uri = "/#{object.class.name.pluralize.underscore}/#{object.id}/avatars/#{object.avatar.id}" unless object.avatar.nil?
```

Comment:
This will need fixing when #946 is merged

Reply:
should be able to replace with `polymorphic_path(object,avatar)`

Reply:
Have not changed. Will wait for 946

Reply:
this can now be changed to
`polymorphic_path([object, object.avatar]) unless object.avatar.nil?`